src/zjs_ocf_server.c

```diff
@@ -150,7 +150,7 @@ struct server_resource *new_server_resource(char *path)
     struct server_resource *resource = zjs_malloc(sizeof(struct server_resource));
     memset(resource, 0, sizeof(struct server_resource));
 
-    resource->resource_path = zjs_malloc(strlen(path));
+    resource->resource_path = zjs_malloc(strlen(path) + 1);
     memcpy(resource->resource_path, path, strlen(path));
     resource->resource_path[strlen(path)] = '\0';
```

Comment:
I think they call this strcpy. :) But we want to use strncpy. Also I don't like the calling strlen 3x in a row on the same thing. And we're actually recommended to use strnlen too.

So go make your life a little more difficult first. :)